.github/workflows/test-connect.yml

```diff
@@ -115,7 +115,7 @@ jobs:
 
   connect-other-devices:
     needs: [build, set-matrix]
-    if: github.event_name == 'schedule' &&  github.repository == 'trezor/trezor-suite'
+    # if: github.event_name == 'schedule' &&  github.repository == 'trezor/trezor-suite'
```

Comment:
yes this should be uncommented

Reply:
Sure. This is under a commit with name [wip: just for dev to make all test run](https://github.com/trezor/trezor-suite/pull/13222/commits/17998ac75da71c3c4784380c60a531b219da4f1b) just for the sake of making sure that all the tests are passing in this PR. It will be removed once the PR is ready to get merged.